base/frontend/sourcegraph-frontend.Service.yaml

```diff
@@ -11,9 +11,8 @@ metadata:
 spec:
   ports:
   - name: http
-    nodePort: 30080
     port: 30080
     targetPort: http
   selector:
     app: sourcegraph-frontend
-  type: NodePort
+  type: ClusterIP
```

Comment:
If people try apply this, won't it fail if the service is already exists (as a NodePort)?

Reply:
I agree, it was a change I wanted to make but did not due to this issue. If so we need to document it in the migration guide. An alternative solution is to update the selector on this service so it matches nothing, and create a new service. I'm unsure which is preferred. Likely the former since an admin upgrading would likely want to be aware of this sort of change (and will need to be doing changes for ingress-nginx anyways).